preprocessing/DataCleanser.py

```diff
@@ -15,7 +20,7 @@ def handleEmptyData(df_covid):
     """
     for col in df_covid.columns:
         if col == 'abstract':
-            df_covid['abstract'].replace('', 'no abstract provided', inplace=True)
+            df_covid['abstract'].replace('', 'Abstract Missing', inplace=True)
```

Comment:
We kept this lowercase to avoid extra work later on when making everything lowercase.